package/MDAnalysis/analysis/encore/clustering/ClusteringMethod.py

```diff
@@ -146,8 +146,10 @@ def __call__(self, distance_matrix):
 
         Returns
         -------
-        numpy.array
-            list of cluster indices
+        numpy.array : array, shape(n_elements) 
+        centroid frames of the clusters for all of the elements
+
+        .. versionchanged:: this method no longer returns details
```

Comment:
`..versionchanged::` should specify the version number for which this function has been modified. Since the next version will be `1.0.0` you should write something like this:
```
.. versionchanged:: 1.0.0
   This method no longer returns ``details``
```